srcpkgs/gamemode/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'gamemode'
+pkgname=gamemode
+version=1.5.1
+revision=1
+build_style=meson
+configure_args="-Dwith-examples=false -Dwith-systemd=false -Dwith-dbus-service-dir=/usr/share/dbus-1/services --prefix=/usr --buildtype debugoptimized"
```

Comment:
Please break long strings at around 80 chars.

Reply:
Most of the args were unnecessary anyways, since they are already set by the meson build helper by default.